packages/management/tkr-conversion-webhook/bundle/config/upstream/certificates.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  labels:
+  name: tkr-conversion-webhook-selfsigned-issuer
+  namespace: tkr-system
```

Comment:
Suggestion(non-blocking): Thinking of using the `#@ data.values.tkrConversionWebhook.namespace` here would simplify it(don't need the overlay file). I understand we used the overlay to modify the namespace in other packages(eg: tkr-infra-machine-webhook), so just thinking that it would be easy and makes easy to make changes and avoid confusion(having to specify the namespace here if it is going to be changed by values.yaml). wdyt?

Reply:
If I understand correctly, we can use the variable from values.yaml inline in this file instead of having an overlay?

So that would mean:
```yaml
    namespace: #@ data.values.tkrConversionWebhook.namespace
```

Reply:
If this is applied to all the entries that are replaced in the overlay, then I think it does make it easier to understand in the future. Someone new coming in and editing this won't be left guessing why they changed the namespace in this file, but it then it's a whole different value when they deploy it.

Reply:
Resolved in f918e3bd435502ee8c71e7856e03fd07d9c9e002